getusers.md

```diff
@@ -16,10 +16,13 @@ We also need a result for each channel. A count from each Group will be fine.
 ## examples, which result i think what we need
 
 ```JSON
-	{
-		"UserID" : "integer",
-    		"ChannelID" : "integer",
-    		"ConnectionID" : "integer",
-	}
+{
+  "from": "String",
+  "to": ["String"],
+  "msg": "String",
```

Comment:
what is msg for?

Reply:
mhh... we don't need this field? but then we don't need the createdAt field, too? I think the get user method should only be for the users to search for an another user how can write him/her